postprocessing/janus-pp-rec.ggo

```diff
@@ -11,3 +11,4 @@ option "videoorient-ext" v "ID of the video-orientation RTP extension (default=n
 option "debug-level" d "Debug/logging level (0=disable debugging, 7=maximum debug level; default=4)" int typestr="1-7" optional
 option "debug-timestamps" D "Enable debug/logging timestamps" flag off
 option "disable-colors" o "Disable color in the logging" flag off
+option "format" f "Specifies the output format (overrides the format from the destination)" string values="opus", "wav", "webm", "mp4", "srt" optional
```

Comment:
Is this syntax for values supported by gengetopt?

Reply:
Looks like it is, and it even does validation for you automatically! That's really cool, I didn't know about this :v:

Reply:
nor did I ;-)